### DISCUSSION TITLE
Is it a stable library?

### DISCUSSION THREAD

**Discussion by kodeplex** [dedupeio/dedupe#1167] [General]
I tried several times to install on windows 10 with latest version of Python and always getting error. Is this library still working? 
I did pip install dedupe, and always getting some strange error on "wheel something".

**Comment by micycle1** [comment]
I had the same problem (Win 11). You need to install build tools to build the wheels.

Download from here:
Webpage: https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022
Direct URL: https://aka.ms/vs/17/release/vs_BuildTools.exe

Make sure these (default selection) are installed:

![image](https://github.com/dedupeio/dedupe/assets/9304234/2816fd83-7f77-41a8-bf84-bff4c5296523)
